style.css

```diff
@@ -743,13 +745,19 @@ footer::after {
 }
 
 @media screen and (max-width: 767px) {
-  .card-categories {
```

Comment:
- [ Optional ] As stated in the project Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider). Kindly adjust it.
- [ Optional ] The Minimum mobile screen size should be 375px.So any Errors in screen sizes less than 375px should be ignored.